cni-plugin/main.go

```diff
@@ -244,6 +246,18 @@ func cmdAdd(args *skel.CmdArgs) error {
 				options.InboundPortsToIgnore = strings.Split(inboundSkipOverride, ",")
 			}
 
+			// Check if there are any subnets to skip
+			subnetSkipOverride, err := getAnnotationOverride(ctx, client, pod, "config.linkerd.io/skip-subnets")
+			if err != nil {
+				logEntry.Errorf("linkerd-cni: could not retrieve overridden annotations: %s", err)
+				return err
+			}
+
+			if subnetSkipOverride != "" {
+				logEntry.Debugf("linkerd-cni: overriding SubnetsToIgnore to %s", subnetSkipOverride)
+				options.SubnetsToIgnore = strings.Split(subnetSkipOverride, ",")
```

Comment:
Thanks for the review. I agree with you that sanitizing to make the user experience better is a good idea, especially since a simple trim would address the issue. I'll amend my PR soon!